test/integration/resultsPage.js

```diff
@@ -18,6 +18,51 @@ function assertSearchResponse(search, assertions) {
     .end(assertions);
 }
 
+function searchNameRanking(res, expected) {
+  const $ = cheerio.load(res.text);
+  const searchResultArr = [];
+  const searchResultBase = $('.results__item--nearby .results__details');
+  searchResultArr.push($('.results__name', searchResultBase.eq(0)).text().trim());
```

Comment:
These 3 lines could be refactored into a single loop, for each item (or the top N (3 in this case)) of the `searchResultBase` array.